### ISSUE TITLE
Overfitting during training expert syncnet

### ISSUE THREAD

**Issue by Welsun** [Rudrabha/Wav2Lip#208]
Hi!  I followed  your instructions and trained the expert syncnet on LRS2. However, after 1000k steps, it started to overfit. The training loss is ~ 0.14, but the eval loss is ~0.5 and still increase. Any suggestions?

**Comment by prajwalkr** [comment]
Save the model with the best validation loss. It is also okay to use an over-fitted model to some extent, as you are only using it as a discriminator.
